### DISCUSSION TITLE
Request for API Endpoint to Access 'Available for Deposit' Data

### DISCUSSION THREAD

**Discussion by DevJyoti0** [etsy/open-api#1290] [Q&A]
I'm working on an application that integrates with the Etsy API, and I need to access the 'Available for Deposit' balance for sellers. I've gone through the Etsy API documentation but couldn't find a specific endpoint or field that provides this data.

Could anyone guide me on how to retrieve 'Available for Deposit' information via the Etsy API? Specifically, I'm looking for:

The correct API endpoint to use.
Which value in the response schema corresponds to the 'Available for Deposit' amount?
If this feature isn't available in the current API, are there any known workarounds or upcoming updates that include this functionality?

Here's the API link that  I've tried so far:
https://openapi.etsy.com/v3/application/shops/{shop_id}/payments
 https://openapi.etsy.com/v3/application/shops/{shop_id}/payment-account/ledger-entries
https://openapi.etsy.com/v3/application/shops/{shop_id}/receipts/{receipt_id}/payments

Any help or pointers would be greatly appreciated!

**Comment by Gareth064** [comment]
Unfortunately if it's not documented in the API, and you're right in this scenario where it isn't, then it's not data you can get to with the API.

**Reply by DevJyoti0** [reply]
Thanks.
